include/networkit/io/NetworkitBinaryWriter.hpp

```diff
@@ -32,6 +42,7 @@ class NetworkitBinaryWriter : public GraphWriter {
 
 	count nodes;
 	count chunks;
+	NetworkitBinaryWeights weightsType;
```

Comment:
This can be stored in a local variable instead of a member variable (see the discussion in https://github.com/networkit/networkit/pull/375#discussion_r302413286).